src/EnergyPlus/UnitVentilator.cc

```diff
@@ -1027,16 +1028,33 @@ namespace UnitVentilator {
                             break;
                         }
                     }
+                    if (!ZoneNodeNotFound) break;
                 }
                 if (ZoneNodeNotFound) {
-                    ShowSevereError(state,
-                                    CurrentModuleObject + " = \"" + state.dataUnitVentilators->UnitVent(UnitVentNum).Name +
-                                        "\". Unit ventilator air inlet node name must be the same as a zone exhaust node name.");
-                    ShowContinueError(state, "..Zone exhaust node name is specified in ZoneHVAC:EquipmentConnections object.");
-                    ShowContinueError(state,
-                                      "..Unit ventilator air inlet node name = " +
-                                          state.dataLoopNodes->NodeID(state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode));
-                    ErrorsFound = true;
+                    bool InletNodeFound = false;
+                    for (CtrlZone = 1; CtrlZone <= state.dataGlobal->NumOfZones; ++CtrlZone) {
+                        if (!state.dataZoneEquip->ZoneEquipConfig(CtrlZone).IsControlled) continue;
+                        for (int NodeNum = 1; NodeNum <= state.dataZoneEquip->ZoneEquipConfig(CtrlZone).NumReturnNodes; ++NodeNum) {
+                            InletNodeFound = ZonePlenum::ValidateInducedNode(state,
+                                                                             state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode,
+                                                                             state.dataZoneEquip->ZoneEquipConfig(CtrlZone).ReturnNode(NodeNum));
+                            if (InletNodeFound) break;
+                        }
+                        if (InletNodeFound) break;
+                    }
+                    if (!InletNodeFound) {
+                        ShowSevereError(
+                            state,
+                            CurrentModuleObject + " = \"" + state.dataUnitVentilators->UnitVent(UnitVentNum).Name +
+                                "\". Unit ventilator air inlet node name must be the same either as a zone exhaust node name or an induce "
+                                "air node in ZoePlenum.");
+                        ShowContinueError(state, "..Zone exhaust node name is specified in ZoneHVAC:EquipmentConnections object.");
+                        ShowContinueError(state, "..Induced Air Outlet Node name is specified in AirLoopHVAC:ReturnPlenum object.");
+                        ShowContinueError(state,
+                                          "..Unit ventilator unit air inlet node name = " +
+                                              state.dataLoopNodes->NodeID(state.dataUnitVentilators->UnitVent(UnitVentNum).AirInNode));
+                        ErrorsFound = true;
+                    }
```

Comment:
Allow UnitVentilator to accept an induced node of return plenum as an inlet node without ATMixer, in addition to a zone exhaust node.